### DISCUSSION TITLE
Reticulum network

### DISCUSSION THREAD

**Discussion by samuk** [named-data/python-ndn#66] [Q&A]
I did a post over on the 

https://reticulum.network/
https://github.com/markqvist/Reticulum

Discussions here: https://github.com/markqvist/Reticulum/discussions/340

I was just wondering if you had anything to add to that? Do the comments there accurately summarise the intent and current state of your project?

**Comment by zjkmxy** [comment]
Sorry but I'm not familiar with Reticulum and I don't know what to reply for now.
